packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -840,6 +834,7 @@ export class MgtPersonCard extends MgtTemplatedComponent {
             <div class="section__title" tabindex=0>${section.displayName}</div>
               <fluent-button
                 appearance="lightweight"
+                aria-label=${this.strings.renderOverviewSectionLabel}
```

Comment:
So, here because we have text content for the button we don't need the aria label as there's already something a screen reader can turn into speech. 
With aria attributes we should only look to use them when they're adding meaning that can't be extracted from the html already.